curriculum/utils.test.ts

```diff
@@ -170,9 +170,11 @@ describe('getSuperOrder', () => {
 });
 
 describe('getSuperBlockFromPath', () => {
-  const directories = fs.readdirSync(
-    path.join(__dirname, './challenges/english')
-  );
+  const junkRegex = /^\./; // Ignore Thumbs.db and .DS_Store
```

Comment:
This won't actually match `Thumbs.db`. Is that intentional?

Reply:
I was so focused on .DS_Store I forgot about it. I'll try the alternative I mentioned.